### PR TITLE
Add package_galaxy_sequence_utils_1_0_1.

### PR DESCRIPTION
**Pull request by blankenberg**
xref https://github.com/galaxyproject/tools-devteam/pull/309
xref https://github.com/galaxyproject/sequence_utils/pull/2

### CODE REVIEW

File: packages/package_galaxy_sequence_utils_1_0_1/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+categories:
+- Tool Dependency Packages
+description: Contains a tool dependency definition that downloads and installs version
+  1.0.1 of the Galaxy sequence utilities.
+long_description: |
+  Galaxy sequence utilities are a set of Python modules for reading,
+  analyzing, and converting sequence formats.
+name: package_galaxy_sequence_utils_1_0_1
+owner: iuc
+remote_repository_url: https://github.com/galaxyproject/tools-devteam/tree/master/packages/package_galaxy_sequence_utils_1_0_1
```
Comment (bgruening) [inline packages/package_galaxy_sequence_utils_1_0_1/.shed.yml]: From URL

Comment (blankenberg) [inline packages/package_galaxy_sequence_utils_1_0_1/.shed.yml]: Fixed the url, thanks!

**Comment by bgruening** [comment]
Thanks Dan!

**Comment by martenson** [comment]
@bgruening This was WIP. Is it done @blankenberg ?

**Comment by blankenberg** [comment]
Just needs https://github.com/galaxyproject/sequence_utils/pull/2 to be merged and the package to become available in depot.

**Pull request merged**